client/src/components/Question.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+export default function Question({handleSubmit, questionChosen, displayQuestion, disabled}) {
+  return (
+    <div>
+      {displayQuestion && 
+      <form onSubmit={handleSubmit}>
+        <p className="question-title">
+          <span className="question-number">{`Question ${questionChosen.id}. `}</span>
+          {questionChosen.title}
+        </p><br/>
+        <input className="placeholder" type="radio" id="choice1" name="choices" value={questionChosen.choices[0]} />
```

Comment:
I think you could simplify this to just use a `questionChosen.choieces.map` to make the code here a little less repetitive.